.gitignore

```diff
@@ -21,6 +21,11 @@
 
 # exclude data from source control by default
 /data/
+cts.csv
+mris.csv
+patients.csv
+xrays.csv
+
```

Comment:
Actually, that above line entirely not quite right. There's no `/data/` folder to exclude in the first place. If I understand correctly where the files would be placed, then something like this would work?
```suggestion
# exclude processed data from source control by default (except the given examples)
notebooks/data/*
!notebooks/data/example*.csv
```